packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -131,7 +134,7 @@ const SquadPage = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [squadId, loggedImpression]);
 
-  const { data: squadMembers } = useQuery<SourceMember[]>(
+  const { data: squadMembers } = useQuery<BasicSourceMember[]>(
     ['squadMembersInitial', handle],
     () => getSquadMembers(squadId),
     { enabled: isBootFetched && !!squadId },
```

Comment:
We should add stale time to this query as well since it will refetch on every window focus or navigating from/to squad page.